src/main/java/eu/arrowhead/blacklist/BlacklistSystemInfo.java

```diff
@@ -18,7 +18,7 @@
 import eu.arrowhead.common.mqtt.model.MqttInterfaceModel;
 import jakarta.annotation.PostConstruct;
 
-@Component
+@Component()
```

Comment:
Why with `()`?

Reply:
Should be Constants.BEAN_NAME_SYSTEM_INFO there. My guess it is because this constant is not merged into the development branch of the common repo.

Still, the nem implementation of management filter creation need this specific name for the systemInfo bean.